mitiq/tests/test_conversions.py

```diff
@@ -97,13 +92,18 @@ def test_to_mitiq(circuit):
     assert input_type in circuit.__module__
 
 
-def test_register_to_mitiq():
-    circuit = qasm_circuit
-    register_mitiq_converter(circuit.__module__, "from", from_qasm)
-    converted_circuit, input_type = convert_to_mitiq(circuit)
-    assert _equal(converted_circuit, cirq_circuit)
-    assert input_type in circuit.__module__
+def test_register_from_to_mitiq(qasm_str=qasm_str, cirq_circuit=cirq_circuit):
```

Comment:
Could remove these since they are global variables.

```suggestion
def test_register_from_to_mitiq():
```